src/pages/dashboardTechnologyPartnerPage/DashboardTablePT.tsx

```diff
@@ -94,6 +94,20 @@ export default function DashboardTablePT({
     setSearchResults(filteredArray);
   };
 
+  const customSort = (a: DelegationResource, b: DelegationResource) => {
+    const aValue = a[orderBy];
+    const bValue = b[orderBy];
+  
+    if (orderBy === 'institutionName' && typeof aValue === 'string' && typeof bValue === 'string') {
+      const compareResult = aValue.localeCompare(bValue, undefined, { sensitivity: 'base' });
+      return order === 'asc' ? compareResult : -compareResult;
+    }
+  
```

Comment:
It will always be institutionName (there is only that)

Checking the string data is not necessary